Makefile

```diff
@@ -25,6 +25,12 @@ proto:
 	for dir in $$(git ls-files '*.proto' | xargs -n1 dirname | uniq); do \
 		protoc -I $$dir --go_out=plugins=grpc:$$dir $$dir/*.proto; \
 	done
+	cat grpclb/grpc_lb_v1/load_balancer.pb.go | \
```

Comment:
> How does go generate solve anything?

`go generate` will allow us to skip the flag to use the grpc plugin for this file.  Generating the grpc services is what introduces the circular dependency.

> Doesn't that contradict your other point about keeping a canonical copy of grpclb.proto?

Yes, but I am looking for practical, short-term options that don't involve `sed`.  Longer-term, if we generate this file with `go generate`, and without the grpc plugin, then it can be a verbatim copy again.  I.e. this option represents a step in the direction of the longer-term goal.